Ports/ja2/package.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=Jagged-Alliance-2
```

Comment:
I think this should always match the directory name:

```suggestion
port=ja2
```